lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -616,6 +623,42 @@ function dosomething_reportback_view_entity($entity) {
   return theme('reportback_permalink', $permalink_vars);
 }
 
+/**
+ * Callback for /reportback/failure page.
+ *
+ * We hit this when a reportback sent to Rogue fails to send back to Phoenix
+ */
+function dosomething_reportback_view_failure() {
+  // Get the user in all the forms that we need
+  global $user;
+  $rb_simple_user = dosomething_user_get_simple_user($rb_user = user_load($user));
+
+
+  // Load the campaign node so we can find out its name, if there's a scholarship, etc.
+  $nid = $_GET['nid'];
```

Comment:
so that part broken and I'm not sure if maybe it never should have worked? Every time I tried it after that initial commit it went to `dosomething_reportback_view_entity` as the callback function